tox.ini

```diff
@@ -4,8 +4,12 @@ envlist = py{310,311,312}{,-notebook}, lint, coverage, docs
 isolated_build = True
 
 [testenv]
-package = wheel
-wheel_build_env = .pkg
+skip_install = false
+allowlist_externals =
+  /bin/bash
+commands_pre =
+  pip install maturin
+  maturin develop --release --manifest-path rust_ext/Cargo.toml
```

Comment:
Double check we need this. I think the rust binaries cant be found during tox testing if it's not here